defaults/main.yml

```diff
@@ -1,3 +1,25 @@
 ---
-namespace: appuio-infra
-openshift_deployment_type: openshift-enterprise
+appuio_pruner_namespace: appuio-pruner
+appuio_pruner_image: "docker.io/appuio/oc:{{ openshift_release }}"
+appuio_pruner_schedule: "@hourly"
+appuio_pruner_state: present
+
+appuio_pruner_cmd:
+  - /usr/bin/oc
+  - adm
+  - prune
+
+appuio_pruner_args:
+  builds:
+    - builds
+    - --orphans
+    - --confirm
+  deployments:
+    - deployments
+    - --orphans
+    - --confirm
+  images:
+    - deployments
+    - --confirm
+
+appuio_pruner_extra_args: {}
```

Comment:
I'd prefer top-level variables, i.e. `appuio_pruner_extra_args_images`. That allows a user to override the arguments for a specific command without having to affect others. Access the variable with `vars['appuio_pruner_extra_args_' ~ item]`.

Reply:
I decided against it because i find the method to access those variables quite ugly, but from a usability point of view it totally makes sense 👍